README.adoc

```diff
@@ -352,23 +352,39 @@ of artists is returned.
 // =================================================================================================
 // Running the Tests
 // =================================================================================================
-[role=command]
-include::{common-includes}/mvnverify.adoc[]
+
+=== Running the tests
+
+Since you started Open Liberty in development mode at the start of the guide, press the `enter/return` key to run the tests.
+
+If the tests pass, you see a similar output to the following example:
 
 [source, role="no_copy"]
 ----
 -------------------------------------------------------
  T E S T S
 -------------------------------------------------------
-Running it.io.openliberty.guides.consumingrest.ConsumingRestTest
-Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.59 sec - in it.io.openliberty.guides.consumingrest.ConsumingRestTest
+Running it.io.openliberty.guides.consumingrest.ConsumingRestIT
+Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.59 sec - in it.io.openliberty.guides.consumingrest.ConsumingRestIT
 
 Results :
 
 Tests run: 4, Failures: 0, Errors: 0, Skipped: 0
 
 ----
 
+When you are done checking out the service, exit development mode by typing `q` in the shell session where you ran the server, 
+and then press the `enter/return` key.
+
+== Building the application
+
+If you are satisfied with your application, run the Maven `package` goal to build the WAR file in the `target` directory:
+
+[role=command]
+----
+mvn package
+----
+
 == Great work! You're done!
 
 You have just accessed a simple RESTful web service and consumed its resources using JSON-B and JSON-P in Open Liberty.
```

Comment:
Change
"You have just accessed a simple RESTful web service and consumed its resources using JSON-B and JSON-P in Open Liberty."
To
"You just accessed a simple RESTful web service and consumed its resources by using JSON-B and JSON-P in Open Liberty."